forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/ApplicationPinDeviceAuthenticator.kt

```diff
@@ -49,7 +50,10 @@ open class ApplicationPinDeviceAuthenticator(private val pinCollector: PinCollec
 
     private val worker = Executors.newSingleThreadScheduledExecutor()
 
-    override suspend fun generateKeys(context: Context): KeyPair {
+    override suspend fun generateKeys(context: Context, attestation: Attestation): KeyPair {
```

Comment:
we dont support attestation for Application PIN ? may be we need to mention in the docs

Reply:
Yes, the following limitation will be required to document

- Key Attestation only support for Android version > 24 (Android N), for device running Android version 23, Key Attestation verification will be skipped.
- Key Attestation is not supported for Application Pin Authentication Type
- Key Attestation is not supported for iOS device

Those are documented in the Design specification and will need to update in the official documentation.